### ISSUE TITLE
mdFabSpeedDial: hidden buttons should not be focusable

### ISSUE THREAD

**Issue by montgomery1944** [angular/material#7771]
Currently in mdFabSpeedDial hidden buttons are focusable, which causes issues with accessibility, when user is using tab to navigate the page, because focus is set to elements not visible to user.

Demo can be found at https://material.angularjs.org/latest/demo/fabSpeedDial.

I can provide a pull request with fixes for this issue, including tests, if this is going to help.

**Comment by ThomasBurleson** [comment]
@topherfangio - duplicate?

**Comment by topherfangio** [comment]
It's related to #5651, but I'm not actually sure that it's a duplicate.

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
        For details, see our [forum posting](http://bit.ly/1UhZyWs).

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
      For details, see our forum posting @ http://bit.ly/1UhZyWs.
